apps/src/templates/rubrics/rubrics.module.scss

```diff
@@ -15,6 +29,11 @@
   border-radius: 30px;
 }
```

Comment:
Would this work to set the opacity on the `img` tag?

```suggestion
  border-radius: 30px;
  
  > img {
    opacity: 1.0;
  }
}
```

Reply:
this looks like it should work, but my first though was just `img {` since it is a direct descendant.

Reply:
That should work. `img {` is less specific than `> img {` which only matches `img` that is a direct child of the button. (but not in any way that overrides the other). CSS is complex. 😓

Reply:
apologies, I had it backwards, you are correct!